.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+
+[*.sh]
+indent_style = space
+indent_size = 2
+
+[*.dts]
+indent_style = tab
+indent_size = 8
```

Comment:
Is this file required for project?

Reply:
Yes. It helps maintain consistent coding styles across various editors and IDE.